### PR TITLE
s390: Improve error reporting for issues with serial output

### PR DESCRIPTION
**Pull request by Soulofdestiny**
Reduces occurence of  sporadic issues with dasdfmt and wait_serial

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
local verification: http://opeth.suse.de/tests/3526

File: tests/installation/bootloader_s390.pm [hunk 1]
```diff
@@ -206,9 +206,9 @@ sub format_dasd() {
     die "dasd_configure died with exit code $r" unless (defined($r) && $r == 0);
 
     # format dasda (this can take up to 20 minutes depending on disk size)
-    $r = script_run("echo yes | dasdfmt -b 4096 -p /dev/dasda", 1200);
-    show_debug();
-    die "dasdfmt died with exit code $r" unless (defined($r) && $r == 0);
+    #$r = script_run("echo yes | dasdfmt -b 4096 -p /dev/dasda", 1200);
```
Comment (coolo) [inline tests/installation/bootloader_s390.pm]: why comment this code?

Comment (Soulofdestiny) [inline tests/installation/bootloader_s390.pm]: leftover from debug, to speed up tests :/

**Comment by okurz** [comment]
Please remind me, why does it improve stability?

**Comment by coolo** [comment]
it doesn't. this was merely thought to give better error reporting in case the serial support dies. Because it lead Matthias to believe there is a problem with the DASD when in reality os-autoinst failed to get the serial from the SUT.

**Comment by okurz** [comment]
then the commit message and PR description is misleading and should be changed

**Comment by Soulofdestiny** [comment]
I agree. I changed summary and commit message

**Comment by Soulofdestiny** [comment]
LGTM by okurz

**Pull request merged**